core/templates/dev/head/components/question-directives/modal-templates/question-editor-modal.directive.html

```diff
@@ -5,7 +5,7 @@ <h3>
 </div>
 <div class="modal-body" style="width: 96%; margin-left: 2%;">
   <question-editor question-id="questionId"
-                   misconceptions="misconceptions"
+                   misconceptions-per-skill="misconceptionsPerSkill"
```

Comment:
Rename to misconceptions-by-skill; ditto elsewhere.

(We typically use that naming format when describing dicts that are formed this way.)

Reply:
Done.